www/_includes/footer_contents.html

```diff
@@ -23,7 +23,7 @@ <h2>General</h2>
                 <h2>Development</h2>
                 <ul class="nav">
                     <li><a target="_blank" href="https://github.com/apache?utf8=%E2%9C%93&amp;q=cordova-">Source Code</a></li>
-                    <li><a target="_blank" href="https://issues.apache.org/jira/browse/CB/">Issue Tracker</a></li>
+                    <li><a target="_blank" href="https://github.com/apache/cordova#filing-a-bug">Issue Tracker</a></li>
```

Comment:
Figured this will be the most appropriate alternative link since we no longer use Jira.